dartsim/BUILD.bazel

```diff
@@ -47,12 +47,11 @@ public_headers = [
 ]
 
 cc_binary(
-    name = "libignition-physics2-dartsim.so",
+    name = "libignition-physics-dartsim-plugin.so",
     srcs = sources + public_headers,
     includes = ["include", "src"],
-    linkopts = ["-Wl,-soname,libignition-physics2-dartsim.so"],
```

Comment:
Tests fail if this line isn't included

Reply:
I don't think that should be the case.  Since this is a dynamically loaded plugin, it shouldn't be linked into the downstream consumers.